### PR TITLE
feat(aerial): Add imagery timaru-2022-2023-0.1m to aerial config file.

### PR DESCRIPTION
**Pull request by linz-basemaps**

### CODE REVIEW

File: config/tileset/aerial.json [hunk 1]
```diff
@@ -660,6 +652,14 @@
       "title": "Timaru 0.075m Urban Aerial Photos (2020)",
       "category": "Urban Aerial Photos"
     },
+    {
+      "2193": "s3://linz-basemaps/2193/timaru_2022-2023_0.1m/01HFASBR6S6H0PKSGJYDF5EN3Z/",
+      "3857": "s3://linz-basemaps/3857/timaru_2022-2023_0.1m/01HFART8EK8PX5B5HYFRXR0G92/",
```
Comment (Wentao-Kuang) [inline config/tileset/aerial.json:657]:
```suggestion
      "2193": "s3://linz-basemaps/2193/timaru_2022-2023_0.1m/01HFASBR6S6H0PKSGJYDF5EN3Z",
      "3857": "s3://linz-basemaps/3857/timaru_2022-2023_0.1m/01HFART8EK8PX5B5HYFRXR0G92",
```

**Comment by dwsilk** [comment]
Remove `timaru-urban-2020-0.075m`, adding this as an individual tileset.

File: config/tileset/aerial.json [hunk 2]
```diff
@@ -428,6 +428,14 @@
       "minZoom": 13,
       "category": "Event"
     },
+    {
```
Comment (Wentao-Kuang) [inline config/tileset/aerial.json:431]: @dwsilk  Do we also add another layer in here?

Comment (dwsilk) [reply config/tileset/aerial.json:431]: https://github.com/linz/basemaps-config/blob/master/config/tileset/aerial.json#L896 <- this is already merged into `master`, so shouldn't be showing as a change on this PR?

**Review by dwsilk — APPROVED** [review]

**Review by dwsilk — APPROVED** [review]

**Pull request merged**